src/cursor.c

```diff
@@ -290,33 +290,32 @@ int Cursor_Free(Cursor *cur) {
   return Cursors_Purge(cur->parent, cur->id);
 }
 
-void Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
+int Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
   CursorList_Lock(cl);
   CursorSpecInfo *info = findInfo(cl, name, NULL);
-  size_t n = 0;
+  if (!info || info->used == 0) {
```

Comment:
@ashtul again I do not think it can be NULL, also why not printing it anyway? why change it and break the ft.info reply?

Reply:
We had `if(info)` before, where we wouldn't (or thought we wouldn't) print all stats.
I'll remove it so we just keep on printing everything as we did